build/CMakeLists.txt

```diff
@@ -143,7 +143,7 @@ ExternalProject_Add(protobuf
     -Dprotobuf_BUILD_TESTS:BOOL=OFF
     -Dprotobuf_WITH_ZLIB:BOOL=OFF
     -Dprotobuf_MSVC_STATIC_RUNTIME:BOOL=OFF
-    -DCMAKE_BUILD_TYPE:STRING=${CMAKE_BUILD_TYPE}
+    -DCMAKE_BUILD_TYPE:STRING=RELEASE
```

Comment:
At line 136 above, can you add:

\# Use -DCMAKE_BUILD_TYPE:STRING=RELEASE to workaround the protobuf build issue
\# described in https://stackoverflow.com/a/60218066/2091555